invenio_users_resources/services/permissions.py

```diff
@@ -30,6 +30,11 @@ class UsersPermissionPolicy(BasePermissionPolicy):
     can_read_email = [IfPublicEmail([AnyUser()], [Self()]), SystemProcess()]
     can_read_details = [Self(), SystemProcess()]
 
+    # Moderation permissions
+    can_block = [SystemProcess()]
```

Comment:
For now, these actions have to be executed with system identity. 
We have a generator in `invenio-administration` but it's not being used consistently. Moreover, that generator was excluded when loading permissions so I went with `SystemProcess` generator for now.